### PR TITLE
feat(agw): Support for EPS bearer context status IE in TAU

### PR DESCRIPTION
**Pull request by pruthvihebbani**
## Title
feat(agw): Support for EPS bearer context status IE in TAU

## Summary
1. Added support to locally deactivate the bearers as per the EPS bearer context status IE received from the test script.
2. Send TAU request with EPS bearer context status IE. 
https://github.com/magma/magma/issues/9195

## Test plan
- Verified sanity suite
- Verified newly added TCs in PR https://github.com/magma/magma/pull/10189

### CODE REVIEW

File: TestCntlrApp/src/enbApp/nb.c [hunk 1]
```diff
@@ -2770,3 +2770,49 @@ PUBLIC S16 NbHandleConfigTai(NbConfigNewTai *configNewTai) {
...
+ * @return  S16
+ *          -# Success : ROK
+ */
+PUBLIC S16 NbEnbRelBearerReqHdl(NbuRelBearerReq *relBearerReq) {
+  NbUeCb *ueCb = NULLP;
+
+  NB_LOG_ENTERFN(&nbCb);
+
+  if (NULLP == relBearerReq) {
+    NB_LOG_ERROR(&nbCb, "Recieved empty(NULL) NbuRelBearerReq");
```
Comment (ulaskozat) [inline TestCntlrApp/src/enbApp/nb.c:2789]: Recieved -> Received

File: TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c [hunk 2]
```diff
@@ -2456,6 +2456,7 @@ PUBLIC S16 nbPrcIncS1apMsg(NbUeCb *ueCb, S1apPdu *pdu, U8 msgType) {
     if (ret != ROK) {
       NB_LOG_ERROR(&nbCb, "Failed to handle S1 handover Cancel Ack");
     }
+      NB_LOG_ERROR(&nbCb, "Failed to handle S1 handover Cancel Ack");
```
Comment (ulaskozat) [inline TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c:2459]: why is this added here?

Comment (ulaskozat) [reply TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c:2459]: did you intend to add a DEBUG log that it was processed successfully?

**Review by ulaskozat — COMMENTED** [review]
added some minor nits. ping me when updated.

File: TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c (hunk 2 above)
Comment (pruthvihebbani) [reply TestCntlrApp/src/enbApp/nb_s1ap_hndlr.c:2459]: I think it got added by mistake while resolving merge conflicts
